packages/freezed/test/json_test.dart

```diff
@@ -120,6 +120,61 @@ Future<void> main() async {
     });
   });
 
+  group('FreezedUnionFallback', () {
+    test('fromJson', () {
```

Comment:
Could you add a test for when there's no `runtimeType` key at all?

Reply:
done